src/components/TodoList/ItemList.tsx

```diff
@@ -0,0 +1,16 @@
+import Button from "../Button/Button";
+import Checkbox from "../Checkbox/Checkbox";
+
+import { Todolist } from "./TodoList";
+
+type Props = { itens: Todolist };
+
+export default function Itemlist({ itens }: Props) {
+  return (
+    <div className="todo-list__grid">
+      <Checkbox />
+      <li className="todo-list__grid--item">{itens.text}</li>
+      <Button variant="cross" color={""} label={""} />
+    </div>

```

Comment:
we have 2 points here
1. The root element is who needs to be the `li` tag, we need to put all content of the row inside of it.
2. at the line **12** `items.text` it's a string so you can put it in a paragraph (`p`) 
3. when you don't need to pass anything in **color** or **label** you should just ignore it and then handle it inside the component (`Button`) - That I see you're already doing this [here](https://github.com/MarcelloAragoni/todo-list/blob/cae304d23a7b42f6e37e93d0d4b41fcb06c0c12b/src/components/Button/Button.tsx#L15)

### Point 3
The button definition should look like this:
```tsx
<Button variant="cross" />
```
You'll only pass what you want, and then your code will check it for you:

**Props**
**variant** is `cross`
**color** is `undefined`

```tsx
 const buttonStyle = cn(
    "button",
    { "button--ghost": variant === "ghost" },
    { "button--cross": variant === "cross" },
    { "button--blue": color === "blue" }
  );
```
then converting it to the values that you had passed:
```tsx
 const buttonStyle = cn(
    "button",
    { "button--ghost": "cross" === "ghost" }, // will NOT apply this class, the conditional is FALSE
    { "button--cross": "cross" === "cross" }, // will apply this class, the conditional is TRUE
    { "button--blue": undefined === "blue" }, // will NOT apply this class, the conditional is FALSE
  );
```